packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -84,34 +80,16 @@ describe('when only one permission matches', () => {
     });
   });
   describe('when it should not match some permission', () => {
-    describe('when `ManageProject` permission is set', () => {
```

Comment:
I always thought it should be fine to have this removed.
It is when we migrate permissions: `ManageProject` maps to `ManageProjectSettings`, `ManageDeveloperSettings` and `ManageProductTypes` as far as I'm concerned. Everything else follows user flows.